### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by PostsDesert**
- Removed question marks from sampleUtterances.txt
- Removed legacy `GMusicWrapper(environ['GOOGLE_EMAIL'], environ['GOOGLE_PASSWORD'])`
- get_artist in music.py will now return more than 5 top tracks

### CODE REVIEW

**Comment by huberf** [comment]
The server currently runs on port 4000 if the port environment variable is not defined. Removing that section of code removes the ability to run on Heroku, and is therefore a major comparability issue. What was your problem with the previous  setup?

**Comment by PostsDesert** [comment]
I had my PORT environment set to 5000. I reset it with `os.environ["PORT"] = '4000'` and now it works find. I'll remove those changes from the PR.

File: speech_assets/sampleUtterances.txt [hunk 1]
```diff
@@ -28,9 +28,9 @@ GeeMusicPlayIFLRadioIntent Play I'm feeling lucky station
 GeeMusicPlayIFLRadioIntent Play my radio station
 GeeMusicPlayIFLRadioIntent Play my station
 
-GeeMusicCurrentlyPlayingIntent What is the current song?
-GeeMusicCurrentlyPlayingIntent What is currently playing?
-GeeMusicCurrentlyPlayingIntent What is playing right now?
-GeeMusicCurrentlyPlayingIntent What song is this?
-GeeMusicCurrentlyPlayingIntent What is the name of this song?
-GeeMusicCurrentlyPlayingIntent What is this?
+GeeMusicCurrentlyPlayingIntent What is the current song
```
Comment (stevenleeg) [inline speech_assets/sampleUtterances.txt]: What's the reasoning behind removing the question marks?

Comment (PostsDesert) [inline speech_assets/sampleUtterances.txt]: When the sampleUtterances are copied into the interaction module it gives you an error
```
Error: There was a problem with your request: The token 'song?' is not valid. Each token must consist only of unicode characters, spaces and valid punctuation marks. Valid punctuation marks are periods for abbreviations, underscores, possessive apostrophes, and hyphens. Occurred in sample 'GeeMusicCurrentlyPlayingIntent What is the current song?' on line 45.
```

Comment (stevenleeg) [inline speech_assets/sampleUtterances.txt]: ohh right! I forgot that I had fixed that in my dashboard but didn't update the text file. Glad you caught that.

**Comment by stevenleeg** [comment]
This looks great. Thanks for the contribution 🍻 !

**Pull request merged**